apps/authentication-service/src/shared/tokenUtils.ts

```diff
@@ -0,0 +1,69 @@
+import fs from "fs";
+import { decode, JwtPayload } from "jsonwebtoken";
+import os from "os";
+import path from "path";
+
+import { ConsoleColors } from "@/shared/enums";
+
+let token: string | undefined;
+const tokenFileName: string = "brkn-arrw.txt";
+
+export const tokenFile: string = path.join(os.tmpdir(), tokenFileName);
+
+export const getLocalToken = (): string | undefined => {
+  logToken(token);
+  return token;
+};
+
+export const setLocalToken = (newToken: string | undefined): void => {
+  token = newToken;
+};
+
+export const setCachedToken = (token: string): boolean => {
+  try {
+    fs.writeFileSync(tokenFile, token);
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
+export const getCachedToken = (): string | undefined => {
+  if (fs.existsSync(tokenFile)) {
+    return fs.readFileSync(tokenFile, "utf8");
+  } else {
```

Comment:
Just preference, but you don't need the else. If it gets after this if, then you can return undefined.